mmv1/third_party/terraform/services/compute/resource_compute_network_firewall_policy_rule_test.go.tmpl

```diff
@@ -525,9 +525,20 @@ resource "google_compute_network_firewall_policy_rule" "fw_policy_rule2" {
 
 func testAccComputeNetworkFirewallPolicyRule_multipleAdd(context map[string]interface{}) string {
 	return acctest.Nprintf(`
+resource "google_compute_network" "network1" {
+  name                    = "tf-test-%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
 resource "google_compute_network_firewall_policy" "fw_policy" {
   name        = "tf-test-policy-%{random_suffix}"
-  description = "Description Update"
+  description = "Resource created for Terraform acceptance testing"
+}
+
+resource "google_compute_network_firewall_policy_association" "fw_policy_a" {
+  name              = "tf-test-policy-a-%{random_suffix}"
```

Comment:
We'll want to test the long form value for the `project` field here, because the DCL will automatically truncate the `projects/` prefix.
```suggestion
  name              = "tf-test-policy-a-%{random_suffix}"
  project            = "projects/%{project_id}"
```